connector/src/main/java/org/geysermc/connector/utils/BlockUtils.java

```diff
@@ -1,257 +1,268 @@
-                outOfWaterButNotOnGround, insideWaterNotOnGround);
-    }
-
-    public static double getSessionBreakTime(GeyserSession session, BlockMapping blockMapping) {
-        PlayerInventory inventory = session.getPlayerInventory();
-        GeyserItemStack item = inventory.getItemInHand();
-        ItemMapping mapping;
-        CompoundTag nbtData;
-        if (item != null) {
-            mapping = item.getMapping(session);
-            nbtData = item.getNbt();
-        } else {
-            mapping = ItemMapping.AIR;
-            nbtData = new CompoundTag("");
-        }
-        return getBreakTime(session, blockMapping, mapping, nbtData, true);
-    }
-
-    /**
-     * Given a position, return the position if a block were located on the specified block face.
-     * @param blockPos the block position
-     * @param face the face of the block - see {@link com.github.steveice10.mc.protocol.data.game.world.block.BlockFace}
-     * @return the block position with the block face accounted for
-     */
-    public static Vector3i getBlockPosition(Vector3i blockPos, int face) {
-        switch (face) {
-            case 0:
-                return blockPos.sub(0, 1, 0);
-            case 1:
-                return blockPos.add(0, 1, 0);
-            case 2:
-                return blockPos.sub(0, 0, 1);
-            case 3:
-                return blockPos.add(0, 0, 1);
-            case 4:
-                return blockPos.sub(1, 0, 0);
-            case 5:
-                return blockPos.add(1, 0, 0);
-        }
-        return blockPos;
-    }
-
-    /**
-     * Taking in a complete Java block state identifier, output just the block ID of this block state without the states.
-     * Examples:
-     * minecraft:oak_log[axis=x] = minecraft:oak_log
-     * minecraft:stone_brick_wall[east=low,north=tall,south=none,up=true,waterlogged=false,west=tall] = minecraft:stone_brick_wall
-     * minecraft:stone = minecraft:stone
-     *
-     * @param fullJavaIdentifier a full Java block identifier, with possible block states.
-     * @return a clean identifier in the format of minecraft:block
-     */
-    public static String getCleanIdentifier(String fullJavaIdentifier) {
-        int stateIndex = fullJavaIdentifier.indexOf('[');
-        if (stateIndex == -1) {
-            // Identical to its clean variation
-            return fullJavaIdentifier;
-        }
-        return fullJavaIdentifier.substring(0, stateIndex);
-    }
-
-    public static BlockCollision getCollision(int blockId, Vector3i blockPos) {
-        BlockCollision collision = Registries.COLLISIONS.get(blockId);
-        if (collision != null) {
-            collision.setPosition(blockPos);
-        }
-        return collision;
-    }
-
-    public static BlockCollision getCollisionAt(GeyserSession session, Vector3i blockPos) {
-        return getCollision(session.getConnector().getWorldManager().getBlockAt(session, blockPos), blockPos);
-    }
-}
+/*

```

Comment:
This large diff is probably because I changed the line encoding from CRLF to LF, which is what I want to try to keep the line encoding as.

Reply:
Strange, I thought git's `core.autocrlf` setting should prevent this from happening. It should be fixed now.